src/main/java/TaskType.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Ensures that these are the only 3 types of tasks.
+ */
+
+public enum TaskType {
```

Comment:
I like the simplicity and structure of this enum, which is set up such that it is easy to add more new TaskTypes in the future if needed.